pkg/terror/error_list.go

```diff
@@ -595,6 +597,7 @@ var (
 	ErrConfigTomlTransform          = New(codeConfigTomlTransform, ClassConfig, ScopeInternal, LevelMedium, "%s")
 	ErrConfigTaskYamlTransform      = New(codeConfigTaskYamlTransform, ClassConfig, ScopeInternal, LevelMedium, "%s")
 	ErrConfigTaskNameEmpty          = New(codeConfigTaskNameEmpty, ClassConfig, ScopeInternal, LevelMedium, "task name should not be empty")
+	ErrConfigTaskNameNotValid       = New(codeConfigTaskNameNotValid, ClassConfig, ScopeInternal, LevelMedium, "task name not valid")
```

Comment:
same order as `codeConfigTaskNameNotValid`?

Reply:
because new code of `codeConfigTaskNameNotValid` must put at last, otherwise it will change other exists error code, but I think similar error in error_list, should be together

Reply:
@amyangfei What's your opinion?

Reply:
I prefer to keep error object the same order with error code, it is easier for developer to maintain the error list. Besides although `ErrConfigTaskNameEmpty` and `ErrConfigTaskNameNotValid` have similar error meanings, they are used in different scenarios and have no more relationships.

Reply:
okay, keep the order [here](https://github.com/pingcap/dm/pull/290/files#diff-9369cf76142d7b377ba0807f57ca0ab3R626) and also in [errors_release.txt](https://github.com/pingcap/dm/pull/290/files#diff-fd0dd41bf163903cf5cc282941cbd3f5R142)